js/main.js

```diff
@@ -0,0 +1,107 @@
+
+// Setting an alert
+var learnMoreButton = document.querySelector("#addToLearnMore");
+learnMoreButton.addEventListener("click", ShowMore);
+
+function ShowMore() {
+  alert("Thanks for finding out more about us");
+}
+
+// Setting an alert 
+var images = document.querySelector("#allImages");
+images.addEventListener("click", seeImages);
+
+function seeImages() {
+  alert("There you go, enjoy all our photos");
+}
+
+// changing background colour
+var boxColor = document.querySelector(".jumbotron");
+boxColor.addEventListener('click', changeColor);
+
+function changeColor() {
+  boxColor.style.backgroundColor = 'yellow';
+
+  var paragraph = document.createElement('p');
+  paragraph.innerText = "SHOW ME MORE JAVASCRIPT";
+
+  boxColor.appendChild(paragraph);
+}
+
+var learnButton = document.querySelector("#addToLearnMore");
+learnButton.addEventListener('click', addParagraph);
+
+function addParagraph() {
+
+  var newParagraph = document.createElement('p');
+  newParagraph.innerText = "SHOW ME MORE We need lots of bike what are the tricks";
+
+  var topOfArticle = document.querySelector('#mainArticles');
+  topOfArticle.appendChild(newParagraph);
+}
+
+var allImagesbutton = document.querySelector("#allImages");
+allImagesbutton.addEventListener("click", showAllImages);
+
+function showAllImages() {
+  var selectingAllimg = document.querySelectorAll("img");
+  for (var i = 0; i < selectingAllimg.length; i++) {
+    selectingAllimg[i].width = selectingAllimg[i].width - 10;
+    selectingAllimg[i].height = selectingAllimg[i].height - 10;
+  }
+}
+
+// Temporary code, testing now. Will need more explanation for this.
+var myButton = document.querySelector("#donateBike");
+
+myButton.addEventListener("click", doSomething);
+
+function doSomething() {
+  var box = document.querySelector(".jumbotron");
+  if (!box.className.includes("red")) {
+    box.className += " yellow";
+  }
+}
+
+
+
+// This works. Posting, Sending (params) to Server
+var request = new XMLHttpRequest(); //creating a request object
```

Comment:
By creating an object twice, only the last implementation will be use. (1st one will be discarded)

Make sure that is what you want, otherwise use the same request object or create a 'postRequest, getRequest'